fixtures/initial_data.json

```diff
@@ -0,0 +1,137 @@
+[
+    {
+        "model": "sites.site",
+        "pk": 8,
+        "fields": {
+            "domain": "tenant-x.local.edly.io",
```

Comment:
Do we really need more than one tenant (site, route, tenant configt) to test the users API?

Reply:
The idea is to check that a user only exists in X tenant, and not in Y.